LFS171x/fabric-material/chaincode/chaincode_example02/chaincode_example02.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Do we use this example anywhere? Can we get rid of this?

Reply:
I believe this is the example we use in "understanding chaincode" section.

Reply:
removed